pom.xml

```diff
@@ -9,11 +9,11 @@
     <relativePath/>  
     <!-- lookup parent from repository --> 
   </parent>  
-  <groupId>com.microsoft</groupId>  
+  <groupId>uk.ac.ox.ctsu</groupId>
   <artifactId>java-base-app</artifactId>  
   <version>0.0.1-SNAPSHOT</version>  
   <name>java-base-app</name>  
-  <description>Base project for Azure Spring Boot</description>  
+  <description>Base project for Spring Boot</description>  
```

Comment:
Spring Boot and Cloud, no? just not **azure** spring cloud.

Reply:
Done